lib/ansible/modules/windows/win_iis_webapplication.ps1

```diff
@@ -13,6 +13,9 @@ $site = Get-AnsibleParam -obj $params -name "site" -type "str" -failifempty $tru
 $state = Get-AnsibleParam -obj $params -name "state" -type "str" -default "present" -validateset "absent","present"
 $physical_path = Get-AnsibleParam -obj $params -name "physical_path" -type "str" -aliases "path"
 $application_pool = Get-AnsibleParam -obj $params -name "application_pool" -type "str"
+$connect_as = Get-AnsibleParam -obj $params -name 'connect_as' -type 'str' -validateset 'specific_user', 'pass_through'
```

Comment:
You should add a default option of pass_through

Reply:
If this were a new module, I would make that the default option, but there could be tasks out there using this module where they have changed the authentication to specific user a different way. Making a default value on this option would break that configuration.
Setting a default also wouldn't be consistent with the behavior of application_pool and physical_path, where they are only set if the user passes them as parameters.

Reply:
you are absolutely right, missed that